### DISCUSSION TITLE
Value Formatting

### DISCUSSION THREAD

**Discussion by elbubi** [ts-korhonen/LibreHardwareMonitorAfterburnerPlugin#7] [Q&A] [answered]
Hi!
First of all, thanks for this developing this much useful plugin.

To the point: Is there any way to alter value formating? Ie: I would like to see fans speed (%) reported without decimals (45% insted of 45.2%)

Regards and thanks in advance!

**Comment by ts-korhonen** [comment, marked as answer]
Hi,
Glad that you find it useful.

Currently there's no way to customize the formatting, but it would certainly be a good feature to add.

**Comment by elbubi** [comment]
Thanks for your inmediate answer!
